### ISSUE TITLE
Firebase 8.3.0 needed

### ISSUE THREAD

**Issue by snorrsi** [GameWorkstore/com.google.firebase.app#2]
dear owner,

It would be greatly appreciated if you could update your repo to include Firebase 8.2.0 for Unity as it was recently released.

best regards,
Snorri

**Comment by Nyankoo** [comment]
+1

**Comment by snorrsi** [comment]
Hi @john-rosa-playkids ,

it would be greatly appreciated if you can update your firebase repositories to include Firebase 8.2.0 as it fixes some Android bugs and more.

**Comment by john-rosa-playkids** [comment]
Hello people, I will start to update today, I hope all of them be available until tomorrow, it let you all know on this thread.

Thank you for always let me know of the new updates!

**Comment by john-rosa-playkids** [comment]
Hi people, all package updates went smoothly! let me know if we can close this issue, Cheers

**Comment by Nyankoo** [comment]
@john-rosa-playkids We're getting the following error when updating the package through UPM:
`Asset Packages/com.google.firebase.app/Firebase/Plugins/x86_64/FirebaseCppApp-8_2_0.so.zip has no meta file, but it's in an immutable folder. The asset will be ignored.`

```
PackageZipExtrator:Expand-Archive : Es wurde kein Positionsparameter gefunden, der das Argument "2020/MyProject/Assets\../Library/Pack
ageCache/com.google.firebase.app@587bc3018b\Firebase\Plugins\x86_64\FirebaseCppApp-8_2_0.so.zip" akzeptiert.
In Zeile:1 Zeichen:1
+ Expand-Archive D:/Development/MyProject/Asse ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Expand-Archive], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Expand-Archive

```

The German text translates to `No positional parameter was found that would accept the argument`.

**Comment by john-rosa-playkids** [comment]
I will be looking into a solution for this until the end of this week.

**Comment by snorrsi** [comment]
> I will be looking into a solution for this until the end of this week. It's a known issue and I am investigating the best way to zip/unzip this file due to the size of the .so file (122MB) and the per file github limit (100MB) ( https://github.com/GameWorkstore/com.google.firebase.app/wiki )

@john-rosa-playkids Yeah that's unfortunate limit. However to solve this .so.zip issue (the zip file works well for me), can't you just create a .so.zip.meta file with some dummy guid to make Unity stop complaining about that meta file missing.

This 8.2.0 version works fine for me.

I see that they have also released Firebase 8.3.0 .. can you release that on github when you have time ?

**Comment by john-rosa-playkids** [comment]
@snorrsi @Nyankoo I decided to remove the file entirely from 8.3.0 and beyond. Installations of this repo should be easy now and don't cause any errors. If you need specific need of Linux 64 bit desktop support, you can still use this repo by adding the .so file on the Assets folder (I recommend something like Plugins/Firebase/x86_64/FirebaseCppApp-X_X_X.so)

Let me know what you think,
Kind Regards,
John

**Comment by Nyankoo** [comment]
@john-rosa-playkids That would be ok for me, as we're not building for these platforms.

**Comment by snorrsi** [comment]
@john-rosa-playkids should work for me, don't use linux for build. Not sure if Unity Cloud uses Linux. Would self include that file rather than having some workaround getting it due to simplicity.

I tried to switch over to using 8.3.0 and noticed that Firebase Functions is missing 8.3.0 tag. Can you add that ?

**Comment by john-rosa-playkids** [comment]
> @john-rosa-playkids should work for me, don't use linux for build. Not sure if Unity Cloud uses Linux. Would self include that file rather than having some workaround getting it due to simplicity.
> 
> I tried to switch over to using 8.3.0 and noticed that Firebase Functions is missing 8.3.0 tag. Can you add that ?

I missed the https://github.com/GameWorkstore/com.google.firebase.functions, just released 8.3.0 there!
